python_base/Dockerfile

```diff
@@ -38,8 +38,8 @@ RUN yum install -y \
         postgresql-devel \
         python-pip \
         python-virtualenv \
-        python-virtualenvwrapper \
-        wget
+        wget \
+        which
```

Comment:
in virtualenvwrapper sadly but it was another half hour to build this locally :(